protocol/messenger.go

```diff
@@ -3704,6 +3704,19 @@ func (m *Messenger) outputToCSV(timestamp uint32, messageID types.HexBytes, from
 	}
 }
 
+func (m *Messenger) shouldSkipDuplicate(messageType protobuf.ApplicationMetadataMessage_Type) bool {
+	// Permit re-processing of ApplicationMetadataMessage_COMMUNITY_DESCRIPTION messages,
+	// as they may be queued pending receipt of decryption keys.
+	allowedDuplicateTypes := []protobuf.ApplicationMetadataMessage_Type{protobuf.ApplicationMetadataMessage_COMMUNITY_DESCRIPTION}
```

Comment:
not that it matters so much, but I would make it a map, with a single message, it doesn't matter, but as we add more, then it becomes O(number of elements) rather than constants